pkg/meta/annotations.go

```diff
@@ -51,29 +51,69 @@ const (
 	// converted from a different version of the API. The API it was converted
 	// from is the value of the annotation.
 	APIConversionAnnotation = "vertica.com/converted-from-api-version"
+
+	// Ignore the cluster lease when doing a revive or start_db.  Use this with
+	// caution, as ignoring the cluster lease when another system is using the
+	// same communal storage will cause corruption.
+	IgnoreClusterLeaseAnnotation = "vertica.com/ignore-cluster-lease"
+
+	// When set to False, this parameter will ensure that when changing the
+	// vertica version that we follow the upgrade path.  The Vertica upgrade
+	// path means you cannot downgrade a Vertica release, nor can you skip any
+	// released Vertica versions when upgrading.
+	IgnoreUpgradePathAnnotation = "vertica.com/ignore-upgrade-path"
+
+	// The timeout, in seconds, to use when the operator restarts a node or the
+	// entire cluster.  If omitted, we use the default timeout of 20 minutes.
+	RestartTimeoutAnnotation = "vertica.com/restart-timeout"
+
+	// Annotations that we add by parsing vertica --version output
+	VersionAnnotation   = "vertica.com/version"
+	BuildDateAnnotation = "vertica.com/buildDate"
+	BuildRefAnnotation  = "vertica.com/buildRef"
+	// Annotation for the database's revive_instance_id
+	ReviveInstanceIDAnnotation = "vertica.com/revive-instance-id"
 )
 
 // IsPauseAnnotationSet will check the annotations for a special value that will
 // pause the operator for the CR.
 func IsPauseAnnotationSet(annotations map[string]string) bool {
-	return lookupBoolAnnotation(annotations, PauseOperatorAnnotation, false)
+	return lookupBoolAnnotation(annotations, PauseOperatorAnnotation)
```

Comment:
Do we still need `AutoRestartVertica` if we have this?

Reply:
Yes. Pause will not run any reconcilers. Where as auto restart vertica will just skilp the restart reconciler.